azure/7.2/ha-port1-mgmt-crosszone-3ports/active.tf

```diff
@@ -21,6 +21,9 @@ resource "azurerm_virtual_machine" "customactivefgtvm" {
   vm_size                      = var.size
   zones                        = [var.zone1]
 
+  delete_os_disk_on_termination    = true
```

Comment:
@mobilesuitzero  not sure on your thoughts here? Might be good to delete all disks after termination. Not sure if customers typically expect that or expect to be able to get their data back if they accidentally destroy...

Reply:
@mobilesuitzero, if the disks remain on a terraform destroy the user will get an error indicating that the Resource Group could not be deleted because there are still resources in the group. 

From a process completion perspective, terraform destroy should be able to remove all the resources it was responsible for creating.

@Joel-Cripps, to your point I would suggest that these deployments are more example than they are production. An unknowing user may attempt the destroy, see the error if disks are not deleted  and not know how to move forward. Resulting in  charges for remaining resources.

Reply:
@Joel-Cripps , when terraform destroy it, it should remove the instances that it created.    But in terms if customer will want to keep the disk or not, that's up to customer, as we can't control the way customer decide to destroy via tf or via the portal.  Hence, it's not sure.

I am fine with or without it either way.  Just as long it can deploy properly.